src/game_state/Level.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LEVEL_H_
+#define LEVEL_H_
+
+#include <memory>
+#include <SFML/Audio.hpp>
+
+#include "GameObject.h"
+#include "Platform.h"
+
+class Level : public GameObject {
+ public:
+  Level(std::shared_ptr<sf::Music> music,
+        std::shared_ptr<std::vector<Platform>> level);
+  ~Level();
+
+  std::shared_ptr<sf::Music> getMusic() { return music; }
+  std::shared_ptr<std::vector<Platform>> getLevel() { return level; }
+
+ private:
+  std::shared_ptr<sf::Music> music;
+  std::shared_ptr<std::vector<Platform>> level;
```

Comment:
alright, but it should be very clear where any GameObject should go though and there has to be an easy way for GameRenderer to iterate through each GameObject from the GameState class